modules/edu.gemini.seqexec.engine/src/test/scala/edu/gemini/seqexec/engine/packageSpec.scala

```diff
@@ -126,7 +125,7 @@ class packageSpec extends FlatSpec {
   it should "be in Running status after starting" in {
     val p = Process.eval(Task.now(start(seqId)))
     val qs = process(p)(qs1).take(1).runLast.unsafePerformSync.get._2
-    assert(qs.sequences(seqId).status === SequenceState.Running)
+    assert(qs.sequences(seqId).status == SequenceState.Running)
```

Comment:
I see.  Scala refuses to do an implicit conversion when there are two options that might apply in scope.  One is the `scalaz.EqualOps` and the other `TripleEquals` I'm not familiar with.  Perhaps there are imports that could be tightened to avoid picking up `TripleEquals`?